content/blog/getting-started-with-k8s-part3/index.md

```diff
@@ -0,0 +1,372 @@
+---
+title: "Getting Started With Kubernetes: Advanced Deployment"
+date: 2020-07-09
+meta_desc: "A tutorial for getting started with Kubernetes using Infrastructure as Code."
+meta_image: getting-started.png
+authors:
+    - sophia-parafina
+tags:
+    - Kubernetes
+
+---
+
+Welcome to the third article in a series using infrastructure as code to deploy applications with Kubernetes. In the previous post, we reviewed basic Kubernetes objects and abstractions used when deploying an application. We examined code examples across the cloud providers to show how to use infrastructure as code to deploy an application using Kubernetes objects. In this installment, we’ll progress from a simple deployment with just a single application container to a complex application with multiple containers and Pods.
+
+<!--more-->
+
+This series walks you through:
+
+- [Building a Kubernetes cluster on cloud providers]({{< relref "/blog/getting-started-with-k8s-part1" >}})
+- [Basic application deployment]({{< relref "/blog/getting-started-with-k8s-part2" >}})
+- Advance application deployment and Helm charts
+- Stateful applications
+- Networking
+- “Day 2” activities such as migrating node groups.
```

Comment:
We also have this already: https://www.pulumi.com/docs/tutorials/kubernetes/eks-migrate-nodegroups/

Reply:
I plan to use it as an example